docs/results.md

```diff
@@ -1,31 +1,22 @@
 # Results
 
-**WIP**
-
-In order to post results properly or receive results emails you must first configure your `~/.ocs-ci.yaml` file.
-Please see the [Initial Setup](#initial-setup) section of the readme if you haven't done that.
-
-## Polarion
-
-**WIP**
-
-Results are posted to polarion if the `--post-results` argument is passed to
-`run.py`. When this argument is used, any tests that have a `polarion-id` 
-configured in the suite will have it's result posted to polarion.
-
 ## Report Portal
 
-**WIP**
+Sending test results to report portal is supported via the [pytest agent](https://github.com/reportportal/agent-python-pytest).
+For in-depth details on how to utilize this plugin, check their documentation.
```

Comment:
@clacroix12 I think we can knock off this line since we are going to handle this.

Reply:
I think it's important to link to their doc because we have the ability to utilize any of the CLI options they provide. We might be handling some of them automatically or in default cases but we shouldn't assume full control since we also want to support posting results from local dev environments.

Reply:
To comment here, Not talking about the link, but just the line 6 to avoid confusion.

Reply:
fixed